docs/data-sources/region.md

```diff
@@ -1,33 +1,49 @@
----
-# generated by https://github.com/hashicorp/terraform-plugin-docs
-page_title: "biganimal_region Data Source - terraform-provider-biganimal"
-subcategory: ""
-description: |-
-  The available regions within a cloud provider
----
-
 # biganimal_region (Data Source)
-
-The available regions within a cloud provider
-
-
+The region data source shows the available regions within a cloud provider.
+
+## Example Usage
+```terraform
+variable "cloud_provider" {
+  type        = string
+  description = "Cloud Provider"
+
+  validation {
+    condition     = contains(["aws", "azure"], var.cloud_provider)
+    error_message = "Please select one of the supported regions: aws, azure."
+  }
+}
+
+data "biganimal_region" "this" {
+  cloud_provider = var.cloud_provider
+  // region_id   = "us-west-1" //optional
+  // query       = "eu" // optional
+}
+
+output "regions" {
+  value = data.biganimal_region.this.regions
+}
+
+output "cloud_provider_id" {
+  value = data.biganimal_region.this.cloud_provider
+}
+```
 
 <!-- schema generated by tfplugindocs -->
 ## Schema
 
 ### Required
 
-- `cloud_provider` (String) Cloud Provider
+- `cloud_provider` (String) Cloud Provider to list the regions. e.g. "aws", "azure".
```

Comment:
```suggestion
- `cloud_provider` (String) Cloud Provider to list the regions. For example, "aws", "azure".
```
We don't use Latin abbreviations and there's a comma afterwards too.

Reply:
I'm changing all the `e.g.`s to `For example,`.

Reply:
Fixed at https://github.com/EnterpriseDB/terraform-provider-biganimal/commit/4de4c956dd0831008781a5f59b9011542e378c5a